database/models/Whitelist.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  DataType,
+  Column,
+  Model,
+  Table,
+  CreatedAt,
+  UpdatedAt,
+} from "sequelize-typescript"
+
+@Table({ tableName: "whitelists" })
+export class Whitelist extends Model {
+  @Column({
+    autoIncrement: true,
+    primaryKey: true,
+    allowNull: false,
+    type: DataType.BIGINT,
+  })
+  id!: number
+
+  @Column({
+    allowNull: false,
+    unique: true,
+    type: DataType.TEXT,
+    validate: {
+      notEmpty: true,
+    },
+  })
+  email!: string
+
+  @Column({
+    allowNull: true,
+    type: DataType.DATE,
+    defaultValue: null,
+  })
+  expiry!: Date
```

Comment:
wrong typing - the exclamation mark means that it can **never** be `null`, which is not what's declared

Reply:
Good catch, https://github.com/isomerpages/isomercms-backend/pull/422/commits/fa8737f8d288a7a00ad4500172dd9d23b9d31a3b

Reply:
the type should be `Date | null` i think

Reply:
Right, fixed in https://github.com/isomerpages/isomercms-backend/pull/422/commits/df92bcbaa153da5cf075fe9e8e0aa43ba2ef18b8